app/src/main/res/layout/subscription_item.xml

```diff
@@ -1,55 +1,97 @@
     xmlns:squareImageView="http://schemas.android.com/apk/de.danoeh.antennapod"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:padding="4dp"
+    android:orientation="vertical"
     android:foreground="?attr/selectableItemBackground">
 
-    <de.danoeh.antennapod.ui.common.SquareImageView
-        android:id="@+id/imgvCover"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/non_square_icon_background"
-        android:scaleType="fitCenter"
-        squareImageView:direction="width"
-        tools:src="@tools:sample/avatars" />
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardOuter"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:clickable="false"
+        app:cardCornerRadius="12dp"
+        app:cardElevation="0dp">
 
-    <com.joanzapata.iconify.widget.IconTextView
-        android:id="@+id/txtvTitle"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_alignStart="@+id/imgvCover"
-        android:layout_alignLeft="@+id/imgvCover"
-        android:layout_alignTop="@+id/imgvCover"
-        android:layout_alignEnd="@+id/imgvCover"
-        android:layout_alignRight="@+id/imgvCover"
-        android:layout_alignBottom="@+id/imgvCover"
-        android:background="@color/non_square_icon_background"
-        android:ellipsize="end"
-        android:gravity="center"
-        android:textColor="?android:attr/textColorPrimary"
-        tools:text="@string/app_name" />
-
-    <de.danoeh.antennapod.ui.common.TriangleLabelView
-        android:id="@+id/triangleCountView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_alignParentTop="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        app:backgroundColor="?attr/colorSecondary"
-        app:corner="rightTop"
-        app:primaryText="Test"
-        app:primaryTextColor="?attr/colorOnSecondary"
-        app:primaryTextSize="12sp" />
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <androidx.cardview.widget.CardView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:clickable="false"
+                app:cardCornerRadius="8dp"
+                app:cardElevation="0dp">
+
+                <RelativeLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+
+                    <de.danoeh.antennapod.ui.common.SquareImageView
+                        android:id="@+id/imgvCover"
```

Comment:
Now that you already re-structure most of this file, it would be great if you could switch to the naming conventions described here: https://github.com/AntennaPod/AntennaPod/wiki/Code-style#resource-id-naming

Something like coverImage, titleLabel, fallbackTitleLabel, etc